00-quickstart/code/sort.py

```diff
@@ -1,3 +1,3 @@
 import sys
 for line in sorted(sys.stdin.readlines()):
-    print line,
+    print(line, end=" ")
```

Comment:
Why add `end=""`?

Reply:
This was the recommendation in the error logs. Not sure why this is preferred.